README.md

```diff
@@ -27,10 +27,10 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="Omotolani Akinmola" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Great job on updating your logo to a custom one! I wanted to offer a suggestion to make your `Readme` file even more professional. If you want to include a custom logo in your project, you'll need to add it to your source directory as an image. However, if you're not planning to include the image, you can simply remove it. 💪
```
Hint: You can include the image in your project using this format: `src="./yourLogoName.jpg"`. 
```
  _For more detailed information, please refer to [this helpful link](https://www.digitalocean.com/community/tutorials/how-to-add-images-to-your-webpage-using-html)._